test/user_management_test.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:openfoodfacts/utils/QueryType.dart';
+import 'package:test/test.dart';
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'test_constants.dart';
+
+void main() {
+  //test('Create user', () async {
```

Comment:
Why is this left commented out?

Reply:
To create a new user we need user_id, email, and password. In both test accounts we have only two of them. 

Also I don't know exactly how the test servers work. When I add a user on them and try the same data again it gives this error: "This username is already taken" but I can imagine that this user gets deleted with every new version which makes the test unsecure if we assume an error response.

I edit the test so that it uses the PROD_USER, from this we can find out the username and the test should always give an error.